fixtures/nodejs-hybrid-app/package.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "nodejs-hybrid-app",
+	"private": true,
+	"scripts": {
+		"dev": "wrangler dev"
+	},
+	"devDependencies": {
+		"@cloudflare/workers-types": "^4.20240222.0",
+		"@types/pg": "^8.11.2",
+		"pg": "^8.11.3",
+		"unenv": "^1.9.0",
```

Comment:
It might just be the way I am debugging it within the workspace but unenv can't find its polyfill files if this is not a local dep.

Reply:
I came across the same issue in my external repro